transactionparser/src/orderbookchanges.js

```diff
@@ -2,6 +2,23 @@ var _ = require('lodash');
 var utils = require('./utils');
 var BigNumber = require('bignumber.js');
 
+var SELL_FLAG = 0x00020000;
```

Comment:
Maybe leave a comment with link to rippled source? https://github.com/ripple/rippled/blob/5d2d88209f1732a0f8d592012094e345cbe3e675/src/ripple/protocol/LedgerFormats.h#L133